subprojects/core/src/integTest/groovy/org/gradle/api/invocation/GradleLifecycleIsolationIntegrationTest.groovy

```diff
@@ -142,7 +142,7 @@ class GradleLifecycleIsolationIntegrationTest extends AbstractIntegrationSpec {
         fails("help")
 
         then:
-        failure.assertHasCause("Failed to isolate 'GradleLifecycle' action: cannot serialize Gradle script object references as these are not supported with the configuration cache.")
+        failure.assertHasDescription("Failed to isolate 'GradleLifecycle' action: cannot serialize Gradle script object references as these are not supported with the configuration cache.")
```

Comment:
🤔 We're failing later, that's why?

Reply:
If I understand you question correctly, then it's more of the other way around. Before, this was a nested cause of an error. Now it's the top-level description of the problem presented to the user